### PR TITLE
quick fix for replicant redmine systemd command

### PR DESCRIPTION
**Pull request by fahlmant**

### CODE REVIEW

File: recipes/app2.rb [hunk 1]
```diff
@@ -192,6 +192,7 @@
     user 'replicant'
     working_directory '/home/replicant/redmine'
     pid_file '/home/replicant/pids/unicorn.pid'
-    exec_start '/home/replicant/.rvm/bin/rvm 1.8.7-p374 exec ./script/server -b 140.211.9.86 -p 8090'
+    exec_start 'RAILS_ENV=production /home/replicant/.rvm/bin/rvm'\
+               '1.8.7-p374 exec ./script/server -b 140.211.9.86 -p 8090'
```
Comment (ramereth) [inline recipes/app2.rb]: Can you make the newline break after ``1.8.7-p374`` please? That makes it much easier to read here. Just make sure you add a space after it.

**Comment by fahlmant** [comment]
@ramereth fixed

**Comment by fahlmant** [comment]
!bump patch ~

**Comment by fahlmant** [comment]
!bump patch ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v1.11.4.  Have a nice day!

**Pull request merged**